test/lsp-mode-test.el

```diff
@@ -102,5 +102,67 @@
   (should (string= "bar" (lsp-resolve-value 'lsp-test-my-var )))
   (should (string= "fn-result" (lsp-resolve-value (-const "fn-result" )))))
 
+(ert-deftest lsp-diagnostics-stats-test ()
+  (let ((workspace (make-lsp--workspace)))
+    (clrhash lsp-diagnostic-stats)
```

Comment:
nit: would it better to `let` bind this `lsp-diagnostic-stats` instead?

Reply:
Yep, fixed.